tests/snapshots/cli__list_mutants_well_tested_examine_name_filter.snap

```diff
@@ -1,6 +1,5 @@
 ---
 source: tests/cli.rs
-assertion_line: 41
```

Comment:
I wonder why this line has been removed. What I did is that I regenerated the snapshot in order to have new/refactored name for it.

Reply:
I don't know!

Reply:
It's probably harmless though.

Reply:
all right, let's leave it then, hopefully, it won't come back.